code/modules/power/cell.dm

```diff
@@ -225,6 +225,7 @@
 	icon_state = "h+cell"
 	maxcharge = 15000
 	chargerate = 2250
+	rating = 2

```

Comment:
Does this cause any side effects?

Reply:
It puts them higher in the ratings for the part switching tool thingy

Reply:
Powercell ratings are determined by charge. There is no need to add ratings to the other cell types, as this is already handled.
https://github.com/tgstation/tgstation/blob/aa8e09ed021228c1e77bffeb28d7f033ab53c702/code/modules/power/cell.dm#L161-L162

Reply:
yeah but the tiers are important to know what level of tech they are at. This is just the cells using them wrong.